src/routes/ethereum-navigator/+page.svelte

```
@@ -0,0 +1,178 @@
+<script lang="ts">
+    import type { PageData } from './$types';
+    import { Menu } from 'lucide-svelte';
```

ESLint

[error] 'Menu' is defined but never used.

(@typescript-eslint/no-unused-vars)